ziti/cmd/ziti/cmd/config_templates/controller.yml

```diff
@@ -93,41 +47,39 @@ edge:
     sessionTimeout: 30m
     # address - required
     # The default address (host:port) to use for enrollment for the Client API. This value must match one of the addresses
-    # defined in a bind point's address field for the `edge-client` API in the web section.
-    address: 127.0.0.1:1280
-  # enrollment - required
-  # A section containing settings pertaining to enrollment.
+    # defined in this webListener's bindPoints.
+    address: {{ .ZitiEdgeCtrlAPI }}
+  # This section is used to define option that are used during enrollment of Edge Routers, Ziti Edge Identities.
   enrollment:
     # signingCert - required
     # A Ziti Identity configuration section that specifically makes use of the cert and key fields to define
     # a signing certificate from the PKI that the Ziti environment is using to sign certificates. The signingCert.cert
     # will be added to the /.well-known CA store that is used to bootstrap trust with the Ziti Controller.
     signingCert:
-      cert: ${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/intermediate.cert.pem
-      key: ${ZITI_SOURCE}/ziti/etc/ca/intermediate/private/intermediate.key.decrypted.pem
+      cert: {{ .ZitiPKI }}/{{ .ZitiSigningIntermediateName }}/certs/{{ .ZitiSigningIntermediateName }}.cert
+      key:  {{ .ZitiPKI }}/{{ .ZitiSigningIntermediateName }}/keys/{{ .ZitiSigningIntermediateName }}.key
     # edgeIdentity - optional
     # A section for identity enrollment specific settings
     edgeIdentity:
-      # duration - optional, default 5m
+      # durationMinutes - optional, default 5m
       # The length of time that a Ziti Edge Identity enrollment should remain valid. After
       # this duration, the enrollment will expire and not longer be usable.
-      duration: 5m
+      duration: 14400m
```

Comment:
default is inconsistent here. it's also probably already defined as a constant that we can now reference and ensure it's correct. can you use the actual const here?

Reply:
Will fix